build.gradle

```diff
@@ -103,6 +108,10 @@ task afterEclipseBuildshipImport(type: Exec) {
 }
 
 
+googleJavaFormat {
+  toolVersion = '1.7'
```

Comment:
Use tabs for indentation in `build.gradle` files, as is already being done elsewhere in this file.